src/createColorValidator.ts

```diff
@@ -0,0 +1,113 @@
+import { BsDiagnostic, Range } from 'brighterscript';
+import { messages } from './plugins/codeStyle/diagnosticMessages';
+import { BsLintRules, RuleColorFormat, RuleColorCase, RuleColorAlpha, RuleColorAlphaDefaults, RuleColorCertCompliant } from './index';
+
+export function createColorValidator(severity: Readonly<BsLintRules>) {
+    const { colorFormat, colorCase, colorAlpha, colorAlphaDefaults, colorCertCompliant } = severity;
+    return (text, range, diagnostics) => {
+        // remove quotes if any exist
+        text = text.replace(/['"]+/g, '');
```

Comment:
In what case do we get quotes? I didn't notice it but this is incredibly wasteful to run that on every literal string of a codebase.